src/Illuminate/Collections/Collection.php

```diff
@@ -918,6 +918,27 @@ public function only($keys)
         return new static(Arr::only($this->items, $keys));
     }
 
+    /**
+     * Get the values for the specified keys from an array.
+     *
+     * @param  \Illuminate\Support\Enumerable<array-key, TKey>|array<array-key, TKey>|string|null  $keys
+     * @return static
+     */
+    public function select($keys)
```

Comment:
Could perhaps use argument unpacking here.

Reply:
I went for consistency with the `only` method... but happy to move to argument unpacking!

Reply:
All good 👍